src/datepicker-input/datepicker-input.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, Input } from "@angular/core";
+
+let nextId = 0;
+
+@Component({
+	selector: "ibm-date-picker-input",
+	template: `
+	<div class="bx--form-item">
+		<div class="bx--date-picker"
+		[ngClass]= "'bx--date-picker--'+type">
```

Comment:
A little space for readability
```suggestion
		[ngClass]= "'bx--date-picker--' + type">
```